app-admin/passwordsafe/passwordsafe-1.11.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake desktop flag-o-matic wxwidgets
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="libressl minimal qr test +xml xvkbd yubikey"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	net-misc/curl
+	sys-apps/util-linux
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	!minimal? ( !!app-misc/pwsafe )
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+RDEPEND="${DEPEND}
+	xvkbd? ( x11-misc/xvkbd )"
+BDEPEND="
+	app-arch/zip
+	sys-devel/gettext
+	test? ( dev-cpp/gtest )"
+
+S="${WORKDIR}/pwsafe-${MY_PV}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.06_beta-system-gtest.patch"
+)
+
+pkg_pretend() {
+	einfo "Checking for -std=c++11 support in compiler"
+	test-flags-CXX -std=c++11 > /dev/null || die
+}
+
+src_prepare() {
+	cmake_src_prepare
+
+	# binary name pwsafe is in use by app-misc/pwsafe, we use passwordsafe
+	# instead. Perform required changes in linking files
+	sed -i install/desktop/pwsafe.desktop -e "s/pwsafe/${PN}/g" || die
+	sed -i docs/pwsafe.1 \
+		-e 's/PWSAFE/PASSWORDSAFE/' \
+		-e "s/^.B pwsafe/.B ${PN}/" || die
+}
+
+src_configure() {
+	setup-wxwidgets
+
+	local mycmakeargs=(
+		-DNO_QR=$(usex !qr)
+		-DNO_GTEST=$(usex !test)
+		-DSYSTEM_GTEST=ON
```

Comment:
Should this really be turned on if `IUSE="-test"` is used?

Reply:
having `SYSTEM_GTEST=ON` or `OFF` has no effect when testing is disabled, so it can be left as-is.
But when testing the testing I discovered a `BDEPEND=dev-cpp/gtest` is necessary!